apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -295,3 +295,14 @@ export function formatPrisonRulingEmailNotification(
 export function stripHtmlTags(html: string): string {
   return html.replace(/(?:<br \/>)/g, '\n').replace(/(?:<\/?strong>)/g, '')
 }
+
+export function formatGender(gender: CaseGender): string {
```

Comment:
Ég útfærði nákvæmlega sama function í https://github.com/island-is/island.is/pull/1900. Ég gerði það í apps/judicial-system/web/src/utils/stepHelper.tsx og þar af leiðandi væri ekki hægt að samnýta hana þannig ég legg til að þessi PR fari inn fyrst og ég byrji að nota þetta function í staðinn.

Ég myndi segja að við ættum samt að hafa retrun gildin í lowercase. Það er ekki 100% víst að við munum aldrei þurfa lowercase gildi út úr þessu þannig ég legg til að skila lowercase og nota `capitalize` sjá https://github.com/island-is/island.is/blob/a867427a27aa47e9e6a91bbd2b3726b0dfb32ac2/libs/judicial-system/formatters/src/lib/formatters.ts#L29 þegar við viljum fá þetta með stórum staf..

Reply:
Sammála.

Reply:
Ah, hélt að hún væri þar.. en já það væri geggjað

Reply:
Done